Framework/PythonInterface/plugins/functions/MsdYi.py

```diff
@@ -1,64 +1,78 @@
-# Mantid Repository : https://github.com/mantidproject/mantid
-#
-# Copyright &copy; 2007 ISIS Rutherford Appleton Laboratory UKRI,
-#     NScD Oak Ridge National Laboratory, European Spallation Source
-#     & Institut Laue - Langevin
-# SPDX - License - Identifier: GPL - 3.0 +
-# pylint: disable=no-init,invalid-name
+#pylint: disable=no-init,invalid-name
 '''
 @author Spencer Howells, ISIS
 @date December 05, 2013
+
+Copyright &copy; 2007-8 ISIS Rutherford Appleton Laboratory, NScD Oak Ridge National Laboratory & European Spallation Source
```

Comment:
To my understanding, we're using the short copyright comment at the beginning of every `.h`, `.cpp` and `.py` file. Why remove it here?